connectors/src/api/slack_channels_linked_with_agent.ts

```diff
@@ -146,7 +144,11 @@ const _patchSlackChannelsLinkedWithAgentHandler = async (
   });
   const joinPromises = await Promise.all(
     slackChannelIds.map((slackChannelId) =>
-      joinChannel(parseInt(connectorId), slackChannelId)
+      launchJoinChannelWorkflow(
```

Comment:
Question: Why one workflow per channel over one workflow taking a list of channels and looping with one activity per channel?